contracts/land/LANDbase.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.4.23;
+
+contract LANDbase {
```

Comment:
http://solidity.readthedocs.io/en/v0.4.24/style-guide.html#contract-and-library-names